api/v1alpha1/jobset_types.go

```diff
@@ -90,6 +90,9 @@ type ReplicatedJobStatus struct {
 
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="Restarts",JSONPath=".status.restarts",type=string,description="Number of restarts"
+// +kubebuilder:printcolumn:name="Completed",type="string",priority=0,JSONPath=".status.conditions[?(@.type==\"Completed\")].status"
+// +kubebuilder:printcolumn:name="Age",JSONPath=".metadata.creationTimestamp",type=date,description="Time this JobSet was created"
```

Comment:
As mentioned in #161 I like the idea of also adding number of completions/failures - but i couldn't find a way to do it using Kubernetes [JSONPath](https://kubernetes.io/docs/reference/kubectl/jsonpath/).

Maybe we could add new fields `Completions`, `Failures` to `JobSetStatus` and make sure to update those during the reconciliation? That will make it easier for us to support completion/failure in printcolumn. Thoughts @ahg-g ?

Reply:
Could you use ReplicatedJobStatus which was added recently? I think they have succeeded but not failed so you could add that

Reply:
We added Ready and Succeeded to ReplicatedJobStatus (see https://github.com/kubernetes-sigs/jobset/blob/main/api/v1alpha1/jobset_types.go#L81), I am not sure though how we can print this per ReplicatedJob?